Curve-finance-suite/Registry-Remove-Pool/src/agent.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  Finding,
+  HandleTransaction,
+  FindingSeverity,
+  FindingType,
+} from 'forta-agent';
+
+import  {
+  provideEventCheckerHandler,
+  FindingGenerator,
+} from 'forta-agent-tools';
+
+import { utils } from 'ethers';
+
+// The signature of the `PoolRemoved` event
+export const REMOVE_POOL_SIGNATURE = "PoolRemoved(address)";
+
+// The Registry interface
+export const R_IFACE: utils.Interface = new utils.Interface([
+  'event PoolRemoved(address indexed pool)'
+]);
+
+// The address of the Curve registry from curve.readthedocs.io
+const REGISTRY_ADDRESS: string = '0x90E00ACe148ca3b23Ac1bC8C240C2a7Dd9c2d7f5';
+
+// Creates a finding generator to be used by provideEventCheckerHandler
+const createFindingGenerator = (alertId: string): FindingGenerator =>
+  (metadata: { [key: string]: any } | undefined): Finding => 
+    Finding.fromObject({
+      name: 'Curve Registry contract called',
```

Comment:
Could change the Finding name to be more specific to the event being emitted rather than simply when the contract is called.

Something like `Pool Removed Event`.